### DISCUSSION TITLE
verify_method_calls

### DISCUSSION THREAD

**Discussion by danlup71** [xunit/xunit#2729] [General]
Are you considering to implement in the future a feature similar to 

Verify_Method_Called(number_calls,method_name,[method_params]) ?

**Comment by bradwilson** [comment]
No.

What you're looking for is generally a feature of a _**mocking framework**_. We don't offer that functionality; we suggest our users get it from third parties (we use [NSubstitute](https://github.com/nsubstitute/NSubstitute) internally for our testing).
